site/_posts/2022-12-22-manageiq-oparin-ga-announcement.md

```diff
@@ -0,0 +1,98 @@
+---
+title: ManageIQ Oparin GA Announcement
+author: Fryguy
+date: 2022-12-22
```

Comment:
Note to self that I have to change this before we merge if it is not released today.